sample/build.gradle

```diff
@@ -142,7 +142,7 @@ dependencies {
     remoteImplementation("com.clevertap.android:clevertap-geofence-sdk:1.2.0")
     remoteImplementation("com.clevertap.android:clevertap-xiaomi-sdk:1.5.2")
     remoteImplementation("com.clevertap.android:push-templates:1.1.0")
-    remoteImplementation("com.clevertap.android:clevertap-hms-sdk:1.3.2")
+    remoteImplementation("com.clevertap.android:clevertap-hms-sdk:1.3.3")
 
     stagingImplementation("com.clevertap.android:clevertap-android-sdk:5.2.1")
```

Comment:
upgrade this as well?

Reply:
Yess. All the sample app changes can be updated in a separated pr? I shall revert this for now